wf-galleon/EAP7-1218_Galleon_Maven_Repository.adoc

```diff
@@ -0,0 +1,74 @@
+= EAP7-1218: Provide a maven repo zip with everything needed to provision EAP using Galleon
+:author:            Jean-Francois Denise
+:email:             jdenise@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Offer the ability to package all Galleon required artifacts to provision WildFly server.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/EAP7-1218[EAP7-1218]
+
+* https://issues.jboss.org/browse/WFGP-132[WFGP-132]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-891[EAP7-891]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+[ ] Engineering
+
+[ ] QE
+
+=== Affected Projects or Components
+
+* OpenShift
+
+=== Other Interested Projects
+
+* WildFly Galleon
+
+== Requirements
+
+In order to address this requirement, we are defining a new WildFly Maven plugin (_org.wildfly.galleon-plugins:wildfly-galleon-maven-plugin_)
+goal named _generate-maven-repository_.
+This goal generates a zipped maven repository from a feature-pack.
+
+When building a WildFly release (-Drelease=true), this plugin will be called to 
+generate /target/wildfly-<version>-galleon-maven-repository.zip file.
+
+This zip file should then be made available from WildFly public releases web site (along with released WildFly).
+
+=== Hard Requirements
+
+* Zip file has a layout compliant with maven local repository. Root dir inside Zip file is named _repository_.
+* Zip file contains binaries and pom.xml files.
+
+=== Nice-to-Have Requirements
+
+=== Non-Requirements
+
+== Implementation Plan
+
+* Add the goal to WildFly galleon-plugins project.
+* Evolve dist/pom.xml to call the plugin.
+
+== Test Plan
+
+== Community Documentation
+We would need to advertise that this zip file exists (release notes?).
```

Comment:
The blog post for the first such release seems reasonable, i.e. https://wildfly.org/news/2019/02/27/WildFly16-Final-Released/

I think that's what you meant. Release notes could also mean something like https://issues.jboss.org/secure/ReleaseNote.jspa?projectId=12315422&version=12340978 but that's not sufficient.

Mentioning it once is fine; after that it's a standard item on the downloads page.

Adding this to wildfly.or/downloads will take some work in the current site (please file a WFLY assigned to me) and on the WIP new site, https://jbossorg.github.io/wildflysite/ (which has a link at the top for filing issues.)